CHANGELOG.md

```diff
@@ -80,6 +80,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Terminal going transparent during visual bell
 - Selection not being cleared when sending chars through a binding
 - Mouse protocols/encodings not being mutually exclusive within themselves
+- Cells reset with template cursor
```

Comment:
Template cursor is an internal implementation detail. We should document the behavior from a user perspective.

Maybe something more like this:

```suggestion
- Cell reset not clearing underline, strikeout and foreground color
```